src/lib/layer.js

```diff
@@ -276,6 +281,7 @@ export default class Layer {
    * @return {Array} - the decoded color
    */
   encodePickingColor(i) {
+    assert(((((i + 1) >> 24) & 255) === 0), `index: ${i} out of coding range`);
```

Comment:
Please change `index: ${i} out of coding range` =>` index out of picking color range`